src/Symfony/Component/VarExporter/Tests/VarExporterTest.php

```diff
@@ -83,6 +83,11 @@ public function provideFailingSerialization()
      */
     public function testExport(string $testName, $value, bool $staticValueExpected = false)
     {
+        if ('datetime' === $testName) {
+            var_dump($value[3]);
+            var_dump(clone $value[3]);
+            var_dump(\Symfony\Component\VarExporter\Internal\Registry::p('DateInterval'));
+        }
```

Comment:
Did you accidentally commit debug code?

Reply:
No that was done on purpose for debugging the failure as I am not able to reproduce it locally.